beekeeper.yml

```diff
@@ -4,23 +4,27 @@ pull_request:
         subtasks:
             - eslint:
                 name: JavaScript lint checks
-                task: beefore-javascript
+                image: pybee/beefore-javascript
             - pycodestyle:
                 name: Python lint checks
-                task: beefore
+                image: pybee/beefore
     - smoke-test:
-        task: batavia-py35
+        image: pybee/batavia-py35
         name: Smoke build (Python 3.5)
         profile: hi-cpu
     - full-test:
         subtasks:
             - py3.6:
                 name: Python 3.6 tests
-                task: batavia-py36
+                image: pybee/batavia-py36
+                critical: False
```

Comment:
From the look of it, the 3.6 tests don't have to be marked as criticial=False - this PR fixes the inconsistencies.

Reply:
Good call.